Makefile

```diff
@@ -37,6 +37,7 @@ build.%: ## Builds the given docker service. Example: make build.datastore
 
 build: ## Builds the all docker setup.
 	$(MAKE) install_node
+	bash ./docker/pre_push_hook.sh --install
```

Comment:
The devs have to execute the command in Windows Subsystem for Linux 2 (WSL2) terminal where the `make` cmd works.

Reply:
We can open an issue for this. But that would require a lot more work such as getting `node` to work locally. etc...

Reply:
Yeah! You are right, I now realize that creating and managing Docker is dependent on a lot of shell scripts. Doing it will add much work and will also add to maintainability. Let's stick with WSL.